website/docs/r/data_forwarding_rule.html.markdown

```diff
@@ -9,25 +9,61 @@ description: |-
 Provider to manage [Sumologic Data Forwarding Rule](https://help.sumologic.com/docs/manage/data-forwarding/amazon-s3-bucket/#forward-datato-s3)
 
 ## Example Usage
+
+For Partitions
 ```hcl
+resource "sumologic_partition" "test_partition" {
+  name               = "testing_rule_partitions"
+  routing_expression = "_sourcecategory=abc/Terraform"
+  is_compliant       = false
+  retention_period   = 30
+  analytics_tier     = "flex"
+}
+
 resource "sumologic_data_forwarding_rule" "example_data_forwarding_rule" {
-    index_id = "00000000024C6155"
-    destination_id = "00000000000732AA"
-    enabled = "true"
-    file_format = "test/{index}/{day}/{hour}/{minute}"
-    payload_schema = "builtInFields"
-    format = "json"
+  index_id = sumologic_partition.test_partition.id
+  destination_id = "00000000000732AA"
+  enabled = true
+  file_format = "test/{index}/{day}/{hour}/{minute}"
+  payload_schema = "builtInFields"
+  format = "json"
+}
+```
+For Scheduled Views
+```hcl
+resource "sumologic_scheduled_view" "failed_connections" {
+  index_name = "failed_connections"
+  query = "_sourceCategory=fire | count"
+  start_time = "2024-09-01T00:00:00Z"
+  retention_period = 1
+  lifecycle {
+    prevent_destroy = true
+    ignore_changes = [index_id]
```

Comment:
This should not needed now?

Reply:
Yeah I have just kept if for the fact that if someone explicitly tries to set index_id = "000000ABCDE..." (some random value) in the resource definition, then with the help of ignore_changes that will be ignored.